docs/administration/deployment/deploying-at-scale/index.md

```diff
@@ -68,18 +68,20 @@ node3   Ready    <none>   97m   v1.18.2
 node4   Ready    <none>   97m   v1.18.2
 ```
 
-**2.** Label your nodes according to the responsibility they should take; `reana.io/system: infrastructure` for infrastructure nodes and `reana.io/system: runtime` for runtime nodes. For example:
+**2.** Label your nodes according to the responsibility they should take; `reana.io/system: infrastructure` for infrastructure nodes, `reana.io/system: runtimebatch` for runtime batch workflow nodes and `reana.io/system: runtimejobs` for runtime job nodes (additionally you can use `reana.io/system: runtimesessions` to split interactive sessions too). For example:
 
 ```console
 $ kubectl label nodes node2 reana.io/system=infrastructure
-$ kubectl label nodes node3 node4 reana.io/system=runtime
+$ kubectl label nodes node3 reana.io/system=runtimebatch
```

Comment:
Any reason why we don't use `node1` here?

Reply:
Because `node1` is the Kubernetes master node where all system pods run.

Reply:
Ah ok, it's like `kind-control-plane` I guess.